examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "Sentinel2-A", SpacecraftActor, [0,0,0], today
</pre><p>For comprehensibility I would suggest to always use name variables in our examples. So</p><p> </p><p><code></code><code>Python</p><p>S2A = ActorBuilder.get_actor_scaffold(</p><p>&nbsp;&nbsp;&nbsp;name="Sentinel2-A", </p><p>   actor_type=paseos.SpacecraftActor, </p><p>   position=[0,0,0], </p><p>   epoch=today</p><p>)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>